src/assets/styles/base.scss

```diff
@@ -29,25 +29,9 @@
        url('../fonts/BCSans/BCSans-BoldItalic.woff') format('woff'); /* Optimized for very modern browsers */
 }
 
-html,
 body {
   min-height: 100vh;
-}
-
-html {
```

Comment:
The 'html' block is only used in the index.html file... after that everything is in 'body'. So this CSS did nothing (decoy pattern).